packages/react/src/button/styles.ts

```diff
@@ -108,8 +112,6 @@ export function buttonStyles({
 		borderWidth: tokens.borderWidth.lg,
 		borderStyle: 'solid',
 		borderRadius: tokens.borderRadius,
-		cursor: 'pointer',
-		fontFamily: tokens.font.body,
```

Comment:
These styles, in addition to `appearance` do not need to exist here as they have already been defined in `BaseButton`